pkg/gce-pd-csi-driver/node.go

```diff
@@ -101,7 +103,12 @@ const (
 	// doc https://cloud.google.com/compute/docs/memory-optimized-machines#x4_disks
 	x4HyperdiskLimit int64 = 39
 	// doc https://cloud.google.com/compute/docs/accelerator-optimized-machines#a4-disks
-	a4HyperdiskLimit     int64 = 127
+	a4HyperdiskLimit int64 = 127
+	// doc https://cloud.google.com/compute/docs/storage-optimized-machines#z3_disks
+	// doc https://cloud.google.com/compute/docs/accelerator-optimized-machines#a3-disks
+	gen3HyperdiskLimit int64 = 31
+	// doc https://cloud.google.com/compute/docs/compute-optimized-machines#h3_disks
+	h3HyperdiskLimit     int64 = 63 // Use limit for Hyperdisk Throughput
```

Comment:
@mattcary Here for H3 machines I am setting the limit for Hd throughput instead of HdB because it seems like it is a more common use case. Please let me know if you think it is appropriate.

Reply:
Hmm, I thought HdT was very niche. I'm looking up usage stats now (unless you already have them?)

Reply:
Discussed offline we will use HdB default instead which is 8-1 = 7

Reply:
sgtm

For future reference, https://cloud.google.com/compute/docs/disks/hyperdisk-perf-limits#max-total-disks-per-vm is not authoritative as there's also per-VM differences like https://cloud.google.com/compute/docs/compute-optimized-machines#disk_and_capacity_limits.